AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -619,20 +623,17 @@ extension MeetingModel: VideoTileObserver {
             } else {
                 videoModel.addRemoteVideoTileState(tileState, completion: {
                     if self.activeMode == .video {
-                        // If the video is not currently being displayed, pause it
-                        if !self.videoModel.isRemoteVideoDisplaying(tileId: tileState.tileId) {
-                            self.currentMeetingSession.audioVideo.pauseRemoteVideoTile(tileId: tileState.tileId)
-                        }
-                        self.videoModel.videoUpdatedHandler?()
+                        self.videoModel.videoSubscriptionUpdatedHandler?()
                     } else {
                         // Currently not in the video view, no need to render the video tile
-                        self.currentMeetingSession.audioVideo.pauseRemoteVideoTile(tileId: tileState.tileId)
+                        self.videoModel.unsubscribeAllRemoteVideos()
```

Comment:
Not sure if I understand the logic here. Previous logic is to pause that tile, but this logic seems to be all tiles?